recipes/netsurf/recipe.sh

```diff
@@ -0,0 +1,42 @@
+VERSION=3.7
+TAR=http://download.netsurf-browser.org/netsurf/releases/source-full/netsurf-all-$VERSION.tar.gz
+BUILD_DEPENDS=(expat curl sdl openssl zlib newlib)
```

Comment:
Normally newlib wouldn't be a build depend, since the build uses the newlib included with the toolchain (perhaps the way the cross-toolchain is handled could be improved). Is it different here for some reason?